core/src/main/scala/caliban/GraphQLRequest.scala

```diff
@@ -13,18 +13,21 @@ case class GraphQLRequest(
   query: Option[String] = None,
   operationName: Option[String] = None,
   variables: Option[Map[String, InputValue]] = None,
-  extensions: Option[Map[String, InputValue]] = None
-) {
+  extensions: Option[Map[String, InputValue]] = None,
+  @transient isHttpGetRequest: Boolean = false
```

Comment:
What does this do in the context of a case class member?

Reply:
This one comes from `jsoniter` (it's not the Scala one). It excludes the case class member from serialization and always uses the default value for deserialization